include/kodlab_mjbots_sdk/common_header.h

```diff
@@ -3,9 +3,11 @@
 #pragma once
 #include <atomic>
 #include <signal.h>  // manage the ctrl+c signal
+#include <type_traits>
+#include <iostream>
 
 namespace kodlab {
-/**
+/*
```

Comment:
small comment error

Reply:
This is an issue with prior code, should be fixed in a seperate, formatting PR.  Likely the entire repository should be formatted to match the stated Drake convention in such a PR.

Reply:
Sure.